File: app/src/main/java/org/dhis2/usescases/teiDashboard/dashboardfragments/indicators/IndicatorsView.java
package org.dhis2.usescases.teiDashboard.dashboardfragments.indicators;

import org.dhis2.data.tuples.Trio;
import org.dhis2.usescases.general.AbstractActivityContracts;
import org.hisp.dhis.android.core.program.ProgramIndicator;

import java.util.List;

import io.reactivex.functions.Consumer;

/**
 * QUADRAM. Created by ppajuelo on 09/04/2019.
 */
public interface IndicatorsView extends AbstractActivityContracts.View {

    void swapIndicators(List<Trio<ProgramIndicator, String, String>> indicators);

}

File: app/src/main/java/org/dhis2/usescases/teidashboard/dashboardfragments/indicators/IndicatorsView.kt
package org.dhis2.usescases.teidashboard.dashboardfragments.indicators

import dhis2.org.analytics.charts.ui.AnalyticsModel
import org.dhis2.usescases.general.AbstractActivityContracts

interface IndicatorsView : AbstractActivityContracts.View {
    fun swapAnalytics(analytics: List<AnalyticsModel>)
}
